open-metadata-implementation/access-services/data-engine/data-engine-server/src/main/java/org/odpi/openmetadata/accessservices/dataengine/server/handlers/DataEngineCollectionHandler.java

```diff
@@ -101,32 +100,32 @@ TransformationProjectBuilder getTransformationProjectBuilder(TransformationProje
      * @throws UserNotAuthorizedException user not authorized to issue this request
      * @throws PropertyServerException    problem accessing the property server
      */
-    public Optional<EntityDetail> findTransformationProjectEntity(String userId, String qualifiedName) throws UserNotAuthorizedException,
+    public Optional<EntityDetail> findCollectionEntity(String userId, String qualifiedName) throws UserNotAuthorizedException,
             PropertyServerException,
             InvalidParameterException {
-        return dataEngineCommonHandler.findEntity(userId, qualifiedName, TransformationProjectMapper.COLLECTION_TYPE_NAME);
+        return dataEngineCommonHandler.findEntity(userId, qualifiedName, CollectionMapper.COLLECTION_TYPE_NAME);
     }
 
     /**
-     * Create CollectionMembership relationships between a Process asset and the Transformation Project. Verifies that the
+     * Create CollectionMembership relationships between a Process asset and a Collection. Verifies that the
      * relationship is not present before creating it
      *
      * @param userId             the name of the calling user
      * @param processGUID        the unique identifier of the process
-     * @param transformationProjectGuid           the unique identifier of the collection that represents the transformation project
+     * @param collectionGUID           the unique identifier of the collection that represents the collection
```

Comment:
Redundant description, I would keep only the first part "the unique identifier of the collection"